static/bidder-params/algorix.json

```diff
@@ -13,6 +13,14 @@
       "type": "string",
       "description": "Your Token",
       "minLength": 1
+    },
+    "placementId": {
+      "type": "string",
+      "description": "An ID which identifies this placement of the impression"
```

Comment:
Do we need a min/max length validation on these fields like we have for token and sid? Without the `minLength` the JSON schema will allow empty values for these fields as well

Reply:
Yes, we don't need min/max length validation, placementId and appId is Optional params. it can be empty value.